test/2-nodes.js

```diff
@@ -131,7 +131,7 @@ describe('basics between 2 nodes', () => {
         expect(msg.data.toString()).to.equal('banana')
         expect(msg.from).to.be.eql(fsB.libp2p.peerInfo.id.toB58String())
         expect(Buffer.isBuffer(msg.seqno)).to.be.true()
-        expect(msg.topicCIDs).to.be.eql(['Z'])
+        expect(msg.topicIDs).to.be.eql(['Z'])
```

Comment:
The protobuf also needs to be updated https://github.com/libp2p/js-libp2p-floodsub/blob/master/src/message/rpc.proto.js

Reply:
I did change it https://github.com/libp2p/js-libp2p-floodsub/pull/49/files#diff-481c9ac76c4779696596262ad13413d8

Reply:
right! Sorry, missed that

Reply:
I did change it, https://github.com/libp2p/js-libp2p-floodsub/pull/49/files#diff-481c9ac76c4779696596262ad13413d8